packages/kit/src/exports/public.d.ts

```diff
@@ -89,13 +99,20 @@ export interface Builder {
 	/** An array of all routes (including prerendered) */
 	routes: RouteDefinition[];
 
+	// TODO 3.0 remove this method
 	/**
 	 * Create separate functions that map to one or more routes of your app.
 	 * @param fn A function that groups a set of routes into an entry point
 	 * @deprecated Use `builder.routes` instead
 	 */
 	createEntries(fn: (route: RouteDefinition) => AdapterEntry): Promise<void>;
 
+	/**
+	 * Find all the assets imported by server files belonging to `routes`
+	 * @param routes
```

Comment:
I don't think this line does much since there's no description and the types are already present below
```suggestion
```